tools/sgyamlfmt/.yamlfmt

```diff
@@ -0,0 +1,3 @@
+formatter:
+  type: basic
+  retain_line_breaks: true
```

Comment:
I added this as a default config. Any thoughts?

Reply:
Good idea! Let's call it `yamlfmt.yaml` so that we get syntax highlighting

Reply:
ah, lets!